pkg/validate/security_context_linux.go

```diff
@@ -710,23 +709,37 @@ var _ = framework.KubeDescribe("Security Context", func() {
 			verifySeccomp(rc, containerID, []string{"grep", "ecc", "/proc/self/status"}, false, "0") // seccomp disabled
 		})
 
-		It("should support seccomp localhost/profile on the container", func() {
+		It("should support seccomp localhost profile on the container", func() {
 			var containerID string
 
 			By("create seccomp sandbox and container")
-			podID, containerID = seccompTestContainer(rc, ic, localhost+blockchmodProfilePath)
+			seccompProfile := &runtimeapi.SecurityProfile{
+				ProfileType:  runtimeapi.SecurityProfile_Localhost,
+				LocalhostRef: blockchmodProfilePath,
+			}
+			podID, containerID = seccompTestContainer(rc, ic, seccompProfile)
 
 			By("verify seccomp profile")
 			verifySeccomp(rc, containerID, []string{"chmod", "400", "/"}, true, "Operation not permitted") // seccomp denied
 		})
 
-		It("should support seccomp default which is unconfined on the container", func() {
+		It("should support seccomp default on the container", func() {
 			var containerID string
-			seccompProfile := ""
+			seccompProfile := &runtimeapi.SecurityProfile{}
 
 			By("create seccomp sandbox and container")
 			podID, containerID = seccompTestContainer(rc, ic, seccompProfile)
 
+			By("verify seccomp profile")
+			verifySeccomp(rc, containerID, []string{"grep", "ecc", "/proc/self/status"}, false, "1") // seccomp enabled
```

Comment:
Trying to update cri-tools in containerd, the containerd CI fails (https://github.com/containerd/containerd/pull/8937#issuecomment-1671407990 / https://github.com/containerd/containerd/actions/runs/5802001382/job/15727585673?pr=8937#step:9:197)
```
    default:   [FAILED] Expected
    default:       <string>: Seccomp:	2
    default: 
    default:   to contain substring
    default:       <string>: 1
```

I have no idea if this test need to be improved or something weird is happening in containerd CI

Reply:
Yes, this needs adoption. See: https://github.com/kubernetes-sigs/cri-tools/pull/1236